src/pages/admin/settings/index.vue

```diff
@@ -40,18 +40,18 @@ meta:
         </div>
       </AppForm>
 
-      <div class="my-6 border-b border-b-primary-40" />
+      <div class="my-8 block border-b border-b-primary-40" />
```

Comment:
I think the `<div>` will be a `block` in any case so not sure this is needed?

Reply:
You're correct; deleted `block`.